classes/class-wc-connect-nux.php

```diff
@@ -290,28 +279,14 @@ public static function get_banner_type_to_display( $status = array() ) {
 		}
 
 		public function get_jetpack_install_status() {
-			// we need to use validate_plugin to check that Jetpack is installed
-			include_once ABSPATH . 'wp-admin/includes/plugin.php';
-
-			// check if jetpack is installed
-			if ( 0 !== validate_plugin( 'jetpack/jetpack.php' ) ) {
-				return self::JETPACK_NOT_INSTALLED;
-			}
-
-			// check if Jetpack is activated
-			if ( ! is_plugin_active( 'jetpack/jetpack.php' ) ) {
-				return self::JETPACK_INSTALLED_NOT_ACTIVATED;
-			}
-
 			if ( defined( 'JETPACK_DEV_DEBUG' ) && true === JETPACK_DEV_DEBUG ) {
```

Comment:
There are some cases where we switch to Offline Mode without the constant set, so it might make sense to use `Status::is_offline_mode()` instead.